### DISCUSSION TITLE
Moving beets.io to GitHub Pages

### DISCUSSION THREAD

**Discussion by jef** [beetbox/beets#3908] [Ideas]
[GitHub Pages](https://pages.github.com/) is awesome. We'd be able to open source the homepage of http://beets.io, it would be TLS compliant by default, and there is minimal DNS changes for the domain registrar.

This could also allow us to use [Material for Mkdocs](https://squidfunk.github.io/mkdocs-material/) in the future (if we wanted) in a easier, automated approach!

**Comment by jackwilsdon** [comment]
It's already hosted on GitHub pages 😋  see the [gh-pages](https://github.com/beetbox/beets/tree/gh-pages) branch. It might be worth linking it somewhere though...

Also: https://beetbox.github.io/beets

**Reply by jef** [reply]
Wow, lol!

Well in that case, perhaps we should enable the https on it!

**Reply by jackwilsdon** [reply]
Yeah, redirecting to HTTPS on the domain seems like a good idea. It looks like [HTTPS is already supported](https://beets.io) but we don't have ["Enforce HTTPS"](https://docs.github.com/en/pages/getting-started-with-github-pages/securing-your-github-pages-site-with-https#enforcing-https-for-your-github-pages-site) turned on.

Is this something you could enable @sampsyo?

**Reply by sampsyo** [reply]
Good call! Done!
